recipes/pandas-datareader/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{%set name = "pandas-datareader" %}
+{%set version = "0.3.0.post0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "1688566dddfba051e64ac193ad325587eecbe423f90ad0b4b104bd1e24a0d750" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - pandas
+    - requests >=2.3.0
+    - requests-file
+    - requests-ftp
+
+test:
+  imports:
+    - pandas_datareader
+
+about:
+  home: https://github.com/pydata/pandas-datareader
+  license: BSD-3
+  license_file: LICENSE.md
+  license_family: BSD
+  summary: 'Up to date remote data access for pandas, works for multiple versions of pandas'
+  dev_url: https://github.com/pydata/pandas-datareader
+  doc_url: https://pandas-datareader.readthedocs.io/en/latest/
+
+extra:
+  recipe-maintainers:
+    - mvanniekerk
```

Comment:
Oops, guessing this was suppose to be you, @mariusvniekerk. Though we should have caught this too. Do you want to send a PR to the feedstock to correct this? Would be happy to merge if you ping me.